nuxeo-coldstorage/src/test/java/org/nuxeo/coldstorage/operations/AbstractTestMoveColdStorageOperation.java

```diff
@@ -127,4 +134,26 @@ public void shouldFailMoveToColdStorageNoContent() throws OperationException, IO
             assertEquals(SC_NOT_FOUND, e.getStatusCode());
         }
     }
+
+    @Test
+    @Deploy("org.nuxeo.coldstorage.test:OSGI-INF/test-thumbnail-recomputation-contrib.xml")
+    @Deploy("org.nuxeo.ecm.platform.thumbnail:OSGI-INF/thumbnail-listener-contrib.xml")
+    @Deploy("org.nuxeo.ecm.platform.thumbnail:OSGI-INF/thumbnail-core-types-contrib.xml")
+    @Deploy("org.nuxeo.ecm.platform.types.api")
+    @Deploy("org.nuxeo.ecm.platform.types.core")
+    public void shouldNotRecomputeThumbnail() throws IOException, OperationException {
+        DocumentModel documentModel = createFileDocument(session, true);
+        Blob originalThumbnail = thumbnailService.getThumbnail(documentModel, session);
+        assertNotNull(originalThumbnail);
+
+        moveContentToColdStorage(session, documentModel);
+
+        transactionalFeature.nextTransaction();
+        documentModel.refresh();
+
+        // Move action will update the main the content with a `null` the thumbnail shouldn't be recomputed
```

Comment:
Thanks for taking into account this point :). could please remove only the comment as it's not accurate